<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bulgarian grammar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="toolTip">
           <string>Show only main word forms</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Main word forms</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Show all word forms</string>
          </property>
          <property name="text">
           <string>All word forms</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="toolTip">
             <string>Select a word from text. 
Select a sinilar word from list.
Click &quot;Add&quot; button to insert in the database. 
New words will apare after program restart.</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Choose what information to see</string>
            </property>
            <item>
             <property name="text">
              <string>Morphology</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Englesh translation</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Thesaurus</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Spell check</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Go to next misspelled word</string>
               </property>
               <property name="text">
                <string>Continue</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="mouseTracking">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Replace</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="toolTip">
             <string>Show the same word in online dictionary.</string>
            </property>
            <property name="text">
             <string>&lt;a href=&quot;http://dic.vanyog.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0068da;&quot;&gt;Online&lt;/span&gt;&lt;/a&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>51</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QListView" name="listView">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Forms of the selected word</string>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="toolTip">
             <string>Information about the selected word form</string>
            </property>
           </widget>
          </widget>
          <widget class="QTextBrowser" name="textBrowser_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Text to process</string>
           </property>
           <property name="styleSheet">
            <string notr="true">selection-color: rgb(52, 12, 255);
selection-background-color: rgb(19, 255, 3);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Choose a grammatical category to find.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <property name="toolTip">
           <string>Choose a value for the grammatical category to find</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="toolTip">
             <string>Click to find in the processed text words with specified grammar property</string>
            </property>
            <property name="text">
             <string>Find</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listView_2">
          <property name="toolTip">
           <string>List of the words from text which is processed</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionAutoCSpellcheck"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Program"/>
    <addaction name="actionDetailed_help"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionDictionary_Info"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_as"/>
    <addaction name="actionOpen_dat_file"/>
    <addaction name="actionExplore_data_directory"/>
    <addaction name="actionDelete_Index_Files"/>
    <addaction name="actionLoad_IDI_Dictionary"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Search"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Spell_check_file"/>
    <addaction name="actionSpell_check_file_UTF_8"/>
    <addaction name="action_Spell_check_the_clipboard"/>
    <addaction name="actionContinue_check"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuSettings"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Program">
   <property name="text">
    <string>About &amp;Program</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save &amp;as</string>
   </property>
  </action>
  <action name="action_Search">
   <property name="text">
    <string>&amp;Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Spell_check_the_clipboard">
   <property name="text">
    <string>&amp;Spell check the clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionContinue_check">
   <property name="text">
    <string>Continue spell checking</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionOpen_dat_file">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open .&amp;dat file</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_Spell_check_file">
   <property name="text">
    <string>Spell check file (Windows 1251)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAutoCSpellcheck">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic clipboard spellcheck</string>
   </property>
  </action>
  <action name="actionDictionary_Info">
   <property name="text">
    <string>Dictionary Info</string>
   </property>
  </action>
  <action name="actionExplore_data_directory">
   <property name="text">
    <string>Explore data directory</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionDelete_Index_Files">
   <property name="text">
    <string>Delete Index Files</string>
   </property>
  </action>
  <action name="actionLoad_IDI_Dictionary">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load IDI Dictionary</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSpell_check_file_UTF_8">
   <property name="text">
    <string>Spell check file (UTF-8)</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDetailed_help">
   <property name="text">
    <string>Detailed help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
